dev/openchpl_data-model.sql

```diff
@@ -2011,6 +2011,7 @@ CREATE TABLE openchpl.surveillance_nonconformity (
 	certification_criterion_id bigint,
 	nonconformity_type varchar(1024),
 	nonconformity_status_id bigint not null,
+	non_conformity_close_date date not null,
```

Comment:
It doesn't look like this is set to "NOT NULL" in the ocd-3572.sql file but i'm not sure if it needs to be removed from here or added there.